irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -171,36 +170,40 @@ namespace {
 
   std::string missRolePerm(
```

Comment:
I would move these three functions from anonymous namespace to `PostgresCommandExecutor` so that `perm_converter` will not be passed in every function call. This looks a little ugly from my point of view :(